packages/@okta/vuepress-site/docs/reference/api/oie-policy-types/index.md

```diff
@@ -2056,3 +2056,141 @@ A Mapping object links together an Application and a Policy.
+
+Earlier, we supported setting recovery factors on the policy. While we will still continue to support it for backwards compatibility of older clients, the new password policy model does not contain recovery factors. This is now moved to the password policy rule model.
+
+### Password Policy Settings Example
+
+```json
+"settings": {
+     "password": {
+       "complexity": {
+         "minLength": 8,
+         "minLowerCase": 1,
+         "minUpperCase": 1,
+         "minNumber": 1,
+         "minSymbol": 0,
+         "excludeUsername": true
+       },
+       "age": {
+         "maxAgeDays": 0,
+         "expireWarnDays": 0,
+         "minAgeMinutes": 0,
+         "historyCount": 0
+       },
+       "lockout": {
+         "maxAttempts": 10,
+         "autoUnlockMinutes": 0,
+         "showLockoutFailures": false
+       }
+     },
+     "delegation": {
+       "options": {
+         "skipUnlock": false
+       }
+     }
+}
+```
+
+The recovery factors are now on the password policy rule configured inside the action `selfServicePasswordReset`. Below are 3 examples of how recovery factors are configured on the rule based on admin requirements.
+
+### Password Policy Rule Action Data
+
+Admin requires end users to verify with just 1 authenticator before they can recover password. Email, Sms, Voice and Okta Verify Push can be used by end users to initiate recovery. No step up authenticators are required.
+
+```json
+"actions": {
+      "passwordChange": {
+        "access": "ALLOW"
+      },
+      "selfServicePasswordReset": {
+        "access": "ALLOW",
+        "requirement": {
+          "primary": {
+            "methods": [
+              "EMAIL",
+              "SMS",
+              "VOICE",
+              "PUSH"
+            ]
+          },
+          "stepUp": {
+            "required": false
+          }
+        }
+      },
+      "selfServiceUnlock": {
+        "access": "ALLOW"
+      }
+}
+```
+
+Admin requires end users to verify 2 authenticators before they can recover password. Email and Okta Verify Push can be used by end users to initiate recovery. Security Question is required as step-up.
+
+```json
+"actions": {
+      "passwordChange": {
+        "access": "ALLOW"
+      },
+      "selfServicePasswordReset": {
+        "access": "ALLOW",
+        "requirement": {
+          "primary": {
+            "methods": [
+              "EMAIL",
+              "PUSH"
+            ]
+          },
+          "stepUp": {
+            "required": true,
+            "methods": [
+              "SECURITY_QUESTION"
+            ]
+          }
+        }
+      },
+      "selfServiceUnlock": {
+        "access": "ALLOW"
+      }
+}
+```
+
+Admin requires end users to verify 2 authenticators before they can recover password. Okta Verify Push can be used by end users to initiate recovery. Any authenticator allowed for MFA/SSO is required as step-up.
```

Comment:
```suggestion
In this example, the requirement is that end users verify two authenticators before they can recover their password. Only Okta Verify Push can be used by end users to initiate recovery. Any authenticator allowed for MFA/SSO is required as step-up.
```

> Any authenticator allowed for MFA/SSO is required as step-up

I'm not sure what this means

Reply:
Maybe something like `A step-up verification is required, for which any enrolled authenticator that can be used for sign-on can be used` makes it more clear?  @jakubvul @AjayThomas-okta

Reply:
Or maybe  `A step-up verification is required, for which any authenticator that can be used for sign-on are available options.`